### ISSUE TITLE
[v3.3.4] Select component throws error when isDisabled is set to true

### ISSUE THREAD

**Issue by shobhitsinghal624** [GeekyAnts/NativeBase#4582]
### Description

When `isDisabled` is set to `true`, the `Select` component throws the error shown below. The error goes away if `_disabled` is set to `{ opacity: 0.8 }`.

### CodeSandBox/Snack link

https://snack.expo.dev/nxfwPCBT2

### NativeBase Version

3.3.4

### Platform

- [ ] Android
- [ ] CRA
- [ ] Expo
- [X] iOS
- [ ] Next

### Additional Information

Note that the snack link does not show the error for some reason, but its clearly visible in XCode.

Sample code:
```
<Select
  minWidth="200"
  placeholder="Select Value"
  isDisabled={true}
  // _disabled={{ opacity: 0.8 }}
>
  <Select.Item label="Option 1" value="option-1" />
  <Select.Item label="Option 2" value="option-2" />
  <Select.Item label="Option 3" value="option-3" />
</Select>
```

Most likely the error is because `opacity` value is set as the string value `'80'` [here](https://github.com/GeekyAnts/NativeBase/blob/0043e80d67193637ace279b7d717003814e5082f/src/theme/components/select.ts#L12).
When I manually override this value to an integer by uncommenting the `_disabled` prop above, the error goes away.

Error message:
```
JSON value '80' of type NSString cannot be converted to NSNumber
```

Error screenshot:
![simulator_screenshot_3EA044BC-9201-4C0A-B2EC-C3E828522ABD](https://user-images.githubusercontent.com/4579093/151200339-d7a0421a-912b-4f47-8945-1ba90bd9fd52.png)

**Comment by ankit-tailor** [comment]
Hi @shobhitsinghal624, I am unable to reproduce the issue, even your snack is also not giving any issue. Can you please check it once?

**Comment by mrcarjul** [comment]
I can confirm/repro the same issue on ipod touch sim, I will use the `_disabled={{ opacity: 0.8 }}` workaround for now

**Comment by gakinson** [comment]
I can also confirm on version 3.2.2

**Comment by ankit-tailor** [comment]
Thanks for reporting the issue. We're looking into it.

**Comment by surajahmed** [comment]
This issue has been fixed in version [3.3.6](https://github.com/GeekyAnts/NativeBase/releases/tag/v3.3.6).
